docs/content/latest/yugabyte-cloud/cloud-develop/_index.md

```diff
@@ -8,7 +8,7 @@ menu:
   latest:
     identifier: cloud-develop
     parent: yugabyte-cloud
-    weight: 25
+    weight: 20
```

Comment:
Headcontent is missing a period : Get started developing applications for Yugabyte Cloud